### DISCUSSION TITLE
Array format not validating unicode markers in json

### DISCUSSION THREAD

**Discussion by IdrissaD** [frictionlessdata/frictionlessdata.io#672] [Q&A] [answered]
I try to validate a csv file against a table schema, and I realized that unicode indicators `u` were breaking the validation of my arrays.

Schema:
```
 {
         "name":"communes",
         "description":"Liste des communes traversées par la randonnée",
         "example":"[{'nom': 'Molezon', 'code_INSEE': '48098'}]",
         "type":"array",
         "items":{
         	"type":"object",
         	"properties":{
         		"nom": {
         			"type":"string"
         		},
         		"code_INSEE":{
         			"type":"integer"
         		}
         	}
         }
      }
```

Value of `communes` field in my file: ```[{u'nom': u'Ispagnac', u'code_INSEE': u'48075'}]```

Error:
```type-error  Type error in the cell "[{u'nom': u'Ispagnac', u'code_INSEE': u'48075'}]" in row "2" and field "communes" at position "9": type is "array/default"```

Link to the workflow that show the errors, at lines 15, 16 and 17 (don't pay attention to the other errors, I fixed it): https://github.com/IdrissaD/schema_randonnee/runs/2381935747?check_suite_focus=true#step:6:15

Is it normal, and is there a way to make this work (apart from removing the `u` with a script) ?

(Edit: thanks for moving it here, didn't see the discussion tab at first...)

**Comment by roll** [comment, marked as answer]
@IdrissaD 
Hi, I think during the data population process `json.dumps` has to be used because currently it's no a valid JSON because of `u` (looks like native Python stringification)

**Reply by IdrissaD** [reply]
Thanks for the tip! Yes it's Python stringification, I don't know why but I thought it was a valid JSON.
